packages/ckeditor5-build-helpjuice-v2/src/hjeditor.ts

```diff
@@ -349,6 +349,18 @@ HelpjuiceEditor.defaultConfig = {
 						`<video controls="" style="max-width: 100%" src="${getUrl}">Your browser does not support HTML5 video.</video>`
 					);
 				}
+			},
+			{
+				name: 'loomProvider',
+				url: new RegExp(`^https:\\/\\/www\\.loom\\.com\\/share\\/(\\w+)`),
```

Comment:
We can use a Regexp literal here:

```suggestion
				url: /^https:\/\/www\.loom\.com\/share\/(\w+)/,
```

Reply:
done, thanks @NicholasJarr